webapp/src/main/java/com/box/l10n/mojito/service/pollableTask/Pollable.java

```diff
@@ -60,4 +60,13 @@
    * @return the delay in seconds
    */
   long timeout() default 3600;
+
+  /**
+   * Indicates if the timeout value set in the annotation should take precedence over the parent's
+   * timeout value.
+   *
+   * @return {@code true} if a parent tasks timeout value should be overridden in the child else
+   *     {@code false} (default)
+   */
+  boolean overrideParentTimeout() default false;
```

Comment:
Can you update the `timeout()` doc to reflect this addition as well something like 

> If the pollable task has a parent task, it will use the parent's timeout unless {@link #overrideParentTimeout()} is to true